e2e/cypress/integration/dcat_us_export.spec.js

```diff
@@ -35,12 +35,12 @@ describe('DCAT-US Export', () => {
     })
 
     // TODO: integrate datajson and usmetadata extensions
-    // it('Can create a zip export of the organization', () => {
-    //     cy.request('/organization/test-organization/draft.json')
-    //     .then((response) => {
-    //         cy.log(response.status, response.body);
-    //     })
-    // })
+    it('Can create a zip export of the organization', () => {
+        cy.request('/organization/test-organization/draft.json')
+        .then((response) => {
+            cy.log(response.status, response.body);
```

Comment:
We probably want to create expectations around this and not just log. Can we expect the response body to contain something?

Reply:
My test wasn't pushed when you reviewed it, but it's [here](https://github.com/GSA/inventory-app/pull/359/files#diff-ab4b110143d4b67e5a8a6f90bc6d4792448cb09ba8204dd74cc575feb365fec6R41-R53)